src/index.tsx

```diff
@@ -4,13 +4,13 @@ import NewsFeed from './NewsFeed/NewsFeed';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
```

Comment:
This is React 17, may be good thing to migrate to 18? there you have another way to render ([old one](https://react.dev/reference/react-dom/render)  vs [new one](https://react.dev/reference/react-dom/client/createRoot)), it is a bit cleaner.